sumy/nlp/tokenizers.py

```diff
@@ -10,6 +10,22 @@
 from .._compat import to_string, to_unicode, unicode
 
 
+class DefaultWordTokenizer(object):
+    def tokenize(self, text):
+        return nltk.word_tokenize(text)
+
+
+class JapaneseWordTokenizer:
+    def tokenize(self, text):
+        try:
+            import tinysegmenter
+        except ImportError as e:
+            print('Please install "tinysegmenter" to use japanese')
```

Comment:
Please don't print anything in the library code. Raise `ValueError("Japanese tokenizer requires tinysegmenter. Please, install it by command 'pip install tinysegmenter'.")` as you can see [for missing numpy](https://github.com/miso-belica/sumy/blob/dev/sumy/summarizers/lex_rank.py#L53). Thanks